asistente_ladm_col/gui/dialogs/dlg_settings.py

```diff
@@ -47,20 +47,20 @@ class SettingsDialog(QDialog, DIALOG_UI):
     db_connection_changed = pyqtSignal(DBConnector, bool, str)  # dbconn, ladm_col_db, source
     active_role_changed = pyqtSignal()
 
-    def __init__(self, parent=None, qgis_utils=None, conn_manager=None, db_source=COLLECTED_DB_SOURCE, tab_pages_list=list(), required_models=list()):
+    def __init__(self, parent=None, qgis_utils=None, conn_manager=None):
         QDialog.__init__(self, parent)
         self.setupUi(self)
         self.logger = Logger()
         self.conn_manager = conn_manager
         self._db = None
         self.qgis_utils = qgis_utils
-        self.db_source = db_source
-        self._required_models = required_models
+        self.db_source = COLLECTED_DB_SOURCE  # default db source
+        self._required_models = []
```

Comment:
Use `list()` instead of `[]`, it's a convention in this plugin.

Reply:
That's right, thank you.